landing_page/app/controllers/microposts_controller.rb

```diff
@@ -1,5 +1,7 @@
 class MicropostsController < ApplicationController
 
+    # before_action :set_micropost, only: [:show, :edit, :update]
```

Comment:
This is dead code and should be removed.

Please note that dead code means a commented line of code, not just because it's a code comment (although I do have restrictions on when to use code comments, but that's another topic 😃)